bundling/bundle-browser.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eu
+
+# Cloudflare Workers don't support top-level await
+sed -i~ 's/^if (isDeno)/if (false)/' ../src/platform.deno.ts
```

Comment:
I don't like this, mostly because regex on source is very ugly and hard to maintain. The whole point is to have one clean file per platform. If we want to support different platforms that require different platform-specific code, we should create a new file and `cp` our way through.

Reply:
can we rewrite to prevent top level awaits?

Reply:
If we provide a new file `platform.web.ts`, we can write it w/o TLA, yes.